### ISSUE TITLE
Bank results misleading, evaluate the internet banking domains instead?

### ISSUE THREAD

**Issue by alexzorin** [gutworth/httpswatch#76]
In almost all cases, banks have brochureware websites with distinct separated internet banking domains/hosts.

Imo it doesn't make much sense to be testing the brochureware endpoint, which is currently all that is tested. Additionally, any potential issues in the actual internet banking section are not going to be uncovered.

i.e.

* ib.nab.com.au (doesn't even listen on 80, but httpswatch reports ' HTTP site doesn't redirect to HTTPS')
* ib.boq.com.au
* ib.mebank.com.au (doesnt listen on 80, misreported by httpswatch)
* online.westpac.com.au (doesnt listen on 80, misreported by httpswatch)

(you get the picture).

The list of banks should either be better curated or have a disclaimer that httpswatch does not actually evaluate the internet banking part of the website, just the brochure part.

Thanks for your work

**Comment by gutworth** [comment]
You are correct. However, one of the goals of HTTPSWatch is to advocate for HTTPS everywhere not just "secure" areas. That is why we mostly link to homepages.

**Comment by sandstrom** [comment]
To avoid many vulnerabilities it's critical that all pages use HTTPS.

For example, a visitor easily gets p0wned by MITM + phishing if the brochureware website is HTTP-only (or without HSTS). I think this issue can be closed.
